src/PowerShellEditorServices/Workspace/ScriptFile.cs

```diff
@@ -215,26 +221,7 @@ internal static List<string> GetLinesInternal(string text)
                 throw new ArgumentNullException(nameof(text));
             }
 
-            // ReadLine returns null immediately for empty string, so special case it.
-            if (text.Length == 0)
-            {
-                return new List<string> {string.Empty};
-            }
-
-            using (var reader = new StringReader(text))
-            {
-                // 50 is a rough guess for typical average line length, this saves some list
-                // resizes in the common case and does not hurt meaningfully if we're wrong.
-                var list = new List<string>(text.Length / 50);
-                string line;
-
-                while ((line = reader.ReadLine()) != null)
-                {
-                    list.Add(line);
-                }
-
-                return list;
-            }
+            return new List<string>(text.Split(s_newlines, StringSplitOptions.None));
```

Comment:
So I take it this splits by `\r\n` if it finds `\r\n` and then splits by `\n` if it just finds `\n`? Just want to make sure that it doesn't get split by `\n` if it finds a `\r\n`